djstripe/models.py

```diff
@@ -320,6 +328,9 @@ class Customer(StripeObject):
     date_purged = models.DateTimeField(null=True, editable=False)
 
     objects = CustomerManager()
+    
+    allow_multiple_subscriptions = ALLOW_MULTIPLE_SUBSCRIPTIONS
+    retain_canceled_subscriptions = RETAIN_CANCELED_SUBSCRIPTIONS
```

Comment:
The motivation here was that, with only a single subscription, the past history was not in fact being retained. For example, if the subscription is cancelled then re-started (maybe with a new plan), the past history is gone because there is only ever one subscription record. So I added the RETAIN_CANCELED_SUBSCRIPTIONS settings so that this behaviour can be approximately replicated when there are multiple subscriptions.